src/components/ResearchGroupCard/style.scss

```diff
@@ -1,23 +1,120 @@
-.App {
-  text-align: center;
+$card-header-font: 24px;
+$card-header-height: 64px;
+
+.card{
+    text-align: center;
+    font-family: 'Futura', sans-serif;
+    background-color: #fafafa;
+    border-style: solid;
+    border-color: #cecece;
+    margin: 10px;
+    height: 450px;
+    min-width: 200px;
+    overflow-wrap: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.card_professor{
```

Comment:
Typically class styles use either `_` or `-`, I see you use a mix of both. Change everything to `-` to keep it consistent, e.g. `.card-professor`. 

Also naming wise, it's better to use generic names that represent the item on the card, e.g. `.card-sub-header` might make more sense.

Reply:
I was going for the convention where you represent a child by parent__child with hyphen-delimited words. But, I messed up by using '_' and not '__'. 

I have now changed the nomenclature to make it consistent with the naming style you have described in this comment and the next one! I have also changed to generic names.